stmt/stmts/account_manage.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2013 The ql Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSES/QL-LICENSE file.
+
+// Copyright 2015 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stmts
+
+import (
+	"github.com/juju/errors"
+	"github.com/pingcap/tidb/context"
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/expression/expressions"
+	"github.com/pingcap/tidb/model"
+	mysql "github.com/pingcap/tidb/mysqldef"
+	"github.com/pingcap/tidb/parser/coldef"
+	"github.com/pingcap/tidb/parser/opcode"
+	"github.com/pingcap/tidb/rset"
+	"github.com/pingcap/tidb/rset/rsets"
+	"github.com/pingcap/tidb/stmt"
+	"github.com/pingcap/tidb/table"
+	"github.com/pingcap/tidb/util/format"
+	"strings"
```

Comment:
It is better to clean import like:
origin package import
empty line
github.com package import